website/docs/rbi.md

```diff
@@ -126,14 +127,17 @@ The most common way to get RBI files for gems is to generate (or regenerate)
 them. To regenerate RBI files in an already-initialized project:
 
 ```bash
-# Update only the autogenerated gem RBIs:
-srb rbi gems
+# Update only the autogenerated gem RBIs for new gems:
+❯ bin/tapioca gems
 
-# Update every kind of RBI file, including autogenerated gem RBIs:
-srb rbi update
+# Update autogenerated gem RBI for a specific gem:
+❯ bin/tapioca gem <gem-name>
+
+# Update autogenerated gem RBIs for all gems:
+❯ bin/tapioca gems --all
 ```
 
-The way this works is that `srb` will load a project's Gemfile and `require`
+The way this works is that `tapioca` will load a project's Gemfile and `require`
 each gem declared within. While loading each gem, it uses runtime reflection to
 learn what things that gem defines, and then serializes this information into a
 single RBI file for that gem.
```

Comment:
Can we add a note here to link to the tapioca docs about how to further customize the `require` process?

Reply:
Done in 7243e6740 👍